controllers/provider-aws/charts/utils-templates/templates/_versions.tpl

```diff
@@ -0,0 +1,83 @@
+{{- define "kubeletcomponentconfigversion" -}}
```

Comment:
This file can be deleted I guess - it's currently not used. We can add it later (with only the content we need) when we require it.

Reply:
Done.